python/ct/cert_analysis/tld_check_test.py

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env python
+# coding=utf-8
+import mock
+import unittest
+from ct.cert_analysis import tld_check
+
+def gen_dns_name(name):
+    dns_name = mock.Mock()
+    dns_name.value = name
+    return dns_name
+
+EXAMPLE = gen_dns_name("example.com")
+
+class TLDCheckTest(unittest.TestCase):
+    def test_tld_list_not_created_until_check_called(self):
+        self.assertIsNone(tld_check.CheckTldMatches.TLD_LIST_)
+        _ = tld_check.CheckTldMatches.check([EXAMPLE], "dNSNames: ")
```

Comment:
Nit: This test will fail when there's no network connectivity. If you really wanted you could mock the tld_list package and have it return a dummy object.